client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -307,9 +320,11 @@ const mapStateToProps = ( state ) => {
 		progressLoaded: getUploadProgressLoaded( state, siteId ),
 		installing: isInstallInProgress( state, siteId ),
 		backPath: getBackPath( state ),
-		showEligibility: ! isJetpack && ( hasEligibilityMessages || ! isEligible ),
+		showEligibility:
+			( ! isJetpack && ( hasEligibilityMessages || ! isEligible ) ) || ! canUploadThemesOrPlugins,
```

Comment:
I don't think we want to show the eligibility warnings at all for Jetpack sites, since that's specific to the AT transfer
```suggestion
		showEligibility:
			( ! isJetpack && ( hasEligibilityMessages || ! isEligible || ! canUploadThemesOrPlugins ) ),
```

Anyway, I don't understand why is this needed. Why the ability to upload themes or plugins affects the AT transfer eligibility?

Reply:
This is to show the eligibility, and this value also presents the user with the correct notice or nudge e.g. "Upgrade to business plan", "This will make your site Atomic" etc.